### DISCUSSION TITLE
General Appreciation Thread

### DISCUSSION THREAD

**Discussion by black-snow** [DependencyTrack/dependency-track#4322] [General]
Just dropped by to leave a big, fat THANKS.

Thanks to all the contributors that not only keep this thing alive but also constantly improve it!
Whenever I have an issue I get a reaction within hours. It's insane. Usually the support here is better than commercial one. (Whether that speaks for this project or ...)

This tool is very much needed and I regret that I haven't even come around to file a PR with a new template for MS Teams :/

**Comment by nscuro** [comment]
Thanks so much @black-snow, you made my day!

Glad you're having a good experience with the support, we're doing our best given the resources at hand. Definitely appreciate your feedback and hope to have you stick around!

Also forwarded this to the team, I'm sure you made a few more people's day with this. :)
